modules/ROOT/pages/bonita-engine-dependencies.adoc

```diff
@@ -1,307 +1,1553 @@
+|  https://www.gnu.org/software/classpath/license.html[GPL2 w/ CPE]
+
+| mysql
+| mysql-connector-java
+| 8.0.14
+| http://dev.mysql.com/doc/connector-j/en/[http://dev.mysql.com/doc/connector-j/en/]
+|  [The GNU General Public License, v2 with FOSS exception]
+
+| org.apache.commons
+| commons-lang3
+| 3.9
+| http://commons.apache.org/proper/commons-lang/[http://commons.apache.org/proper/commons-lang/]
+|  https://www.apache.org/licenses/LICENSE-2.0.txt[Apache License, Version 2.0]
+
+| org.postgresql
+| postgresql
+| 42.2.5
+| https://jdbc.postgresql.org/[https://jdbc.postgresql.org/]
+|  http://www.postgresql.org/about/licence/[The PostgreSQL License]
+
+| org.slf4j
+| slf4j-api
+| 1.7.25
+| http://www.slf4j.org[http://www.slf4j.org]
+|  http://www.opensource.org/licenses/mit-license.php[MIT License]
+
+| org.springframework
+| spring-aop
+| 5.2.2.RELEASE
+| https://github.com/spring-projects/spring-framework[https://github.com/spring-projects/spring-framework]
+|  https://www.apache.org/licenses/LICENSE-2.0[Apache License, Version 2.0]
+
+| org.springframework
+| spring-beans
+| 5.2.2.RELEASE
+| https://github.com/spring-projects/spring-framework[https://github.com/spring-projects/spring-framework]
+|  https://www.apache.org/licenses/LICENSE-2.0[Apache License, Version 2.0]
+
+| org.springframework
+| spring-context
+| 5.2.2.RELEASE
+| https://github.com/spring-projects/spring-framework[https://github.com/spring-projects/spring-framework]
+|  https://www.apache.org/licenses/LICENSE-2.0[Apache License, Version 2.0]
+
+| org.springframework
+| spring-core
+| 5.2.2.RELEASE
+| https://github.com/spring-projects/spring-framework[https://github.com/spring-projects/spring-framework]
+|  https://www.apache.org/licenses/LICENSE-2.0[Apache License, Version 2.0]
+
+| org.springframework
+| spring-expression
+| 5.2.2.RELEASE
+| https://github.com/spring-projects/spring-framework[https://github.com/spring-projects/spring-framework]
+|  https://www.apache.org/licenses/LICENSE-2.0[Apache License, Version 2.0]
+
+| org.springframework
+| spring-jcl
+| 5.2.2.RELEASE
+| https://github.com/spring-projects/spring-framework[https://github.com/spring-projects/spring-framework]
+|  https://www.apache.org/licenses/LICENSE-2.0[Apache License, Version 2.0]
+
+| org.springframework
+| spring-jdbc
+| 5.2.2.RELEASE
+| https://github.com/spring-projects/spring-framework[https://github.com/spring-projects/spring-framework]
+|  https://www.apache.org/licenses/LICENSE-2.0[Apache License, Version 2.0]
+
+| org.springframework
+| spring-tx
+| 5.2.2.RELEASE
+| https://github.com/spring-projects/spring-framework[https://github.com/spring-projects/spring-framework]
+|  https://www.apache.org/licenses/LICENSE-2.0[Apache License, Version 2.0]
+
+| org.springframework.boot
+| spring-boot
+| 2.2.2.RELEASE
+| https://projects.spring.io/spring-boot/#/spring-boot-parent/spring-boot[https://projects.spring.io/spring-boot/#/spring-boot-parent/spring-boot]
```

Comment:
You must escape all # used in blocs, else it highlights some stuff
see ex 7 in https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/#text-formatting

Reply:
notice that it seems we don't have the issue in 7.12, see #1480

Reply:
@alachambre Reading the docs, I don't understand How I am supposed to escape those # chars to have them simply display normally 🤔

Reply:
https://github.com/bonitasoft/bonita-doc/commit/869982f95ea3689ef6d9766d2857d6d82c6d5a07

Reply:
Thx, it should be fixed now